specificatie/openapi.yaml

```diff
@@ -77,7 +77,14 @@ paths:
       tags:
         - Adres
       summary: "vindt adressen"
-      description: "Vind een actueel adres met de identificatie van het zoekresultaat uit de operatie get /adressen/zoek, de pandidentificatie of de adresseerbaarobjectidentificatie. Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/fields.feature). Gebruik de expand-parameter als je het antwoord wil uitbreiden met (delen van) de gerelateerde resources nummeraanduiding, woonplaats en openbare ruimte, zie [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/expand.feature)."
+      description: >- 
```

Comment:
is dit bewust gedaan om de vorm van opnemen description te wijzigen?

hier zat description in aanhalingstekens en is dat nu met >- gedaan. In de genereervariant zijn juist aanhalingstekens toegevoegd

Reply:
Dit was n.a.v. een opmerking van Melvin om documentatie in markdown op te nemen zodat je niet zoveel horizontaal hoeft te scrollen. Dit maakt de originele yaml iets leesbaarder. Bij het genereren van de geresolvede variant wordt de tekst tussen " " gezet en als je die versie in swagger opent is de documentatie weer compact, evt. formattering in de tekst blijft dan wel bewaard.

Reply:
In de andere Haal Centraal API's gebruik ik daar | (pipe-teken) voor. Dat doet volgens mij hetzelfde als >- ( Aangeven dat wat volgt Markdown is. )